OrbitCore/CaptureData.h

```diff
@@ -64,8 +64,8 @@ class CaptureData {
     return thread_names_;
   }
 
-  [[nodiscard]] const std::string& GetThreadName(int32_t thread_id) const {
-    return thread_names_.at(thread_id);
+  [[nodiscard]] const std::string& GetThreadName(int32_t thread_id) {
+    return thread_names_[thread_id];
```

Comment:
You can keep it const by doing find() -> return ""; in case it was not found. No need to modify the map.

Reply:
Then we'd have to change the return type. I guess using a static empty string would do the trick.